AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.html

```diff
@@ -13,6 +13,7 @@
           type="date"
           formControlName="fromDate"
           name="fromDate"
+          min="{{ todayDate }}"
```

Comment:
Is there a reason you're not using the date pipe here?

Reply:
no particular reason. I use a string as I need to use the same string in different places. I thought I can use the same string for the minimum and for filling the input box with a today date. See line 88.

Reply:
I have now refactored the code using date pipe in the html